main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -91,11 +95,11 @@ public void validate(NoticeContainer noticeContainer) {
    *     separates them is less or equal than {@code
    *     ShapeIncreasingDistanceValidator#MAX_DISTANCE_SHAPEPOINTS}; false otherwise.
    */
-  private static boolean areCloseWithDifferentCoordinates(GtfsShape shape, GtfsShape otherShape) {
+  private static boolean areFarWithDifferentCoordinates(GtfsShape shape, GtfsShape otherShape) {
     return shape.shapePtLon() != otherShape.shapePtLon()
         && shape.shapePtLat() != otherShape.shapePtLat()
```

Comment:
You can have different coordinates while either lon XOR lat is identical. Shouldn't this be `!(shape.shapePtLon() == otherShape.shapePtLon() && shape.shapePtLat() == otherShape.shapePtLat())`

Reply:
> Shouldn't this be !(shape.shapePtLon() == otherShape.shapePtLon() && shape.shapePtLat() == otherShape.shapePtLat())

Thanks for flagging that! Modified in https://github.com/MobilityData/gtfs-validator/pull/1083/commits/574cd7b1ec7691621755ab7b34892ded7bbf6462.